tests/client/amo/components/TestScreenShots.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { renderIntoDocument } from 'react-addons-test-utils';
+import { PhotoSwipeGallery } from 'react-photoswipe';
+
+import ScreenShots from 'amo/components/ScreenShots';
+import { shallowRender } from 'tests/client/helpers';
+
+describe('<ScreenShots />', () => {
+  const onePxImg = 'data:image/gif;base64,R0lGODlhAQABAIAAAAUEBAAAACwAAAAAAQABAAACAkQBADs=';
```

Comment:
Also this is only used in one test, it could go into there as it really only relates to it (for now, anyway).